src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/LinkWidget.tsx

```diff
@@ -108,7 +108,9 @@ class DeletableLinkWidget extends DefaultLinkWidget {
         onMouseEnter={() => this.setState({ selected: true })}
         onClick={() => {
           link.remove()
-          this.props.diagramEngine.repaintCanvas()
+
+          const diagramEngine = (this.props.diagramEngine as ExtendedDiagramEngine)
```

Comment:
Those parentheses are superfluous and should have been removed automatically by TSLint/Prettier. Can you check if they are correctly configured?

Reply:
Yes they are! Prettier was somehow disabled on my side!